### ISSUE TITLE
Logging system for GUI2

### ISSUE THREAD

**Issue by somebody1234** [enso-org/enso#8567]
GUI2 will eventually need a logging system... the primary usecase being sending diagnostic logs to the cloud.
- Related: https://github.com/enso-org/cloud-v2/issues/735

At the bare minimum this probably needs to have logging levels (the usual debug/info/warn/error/panic)

**Comment by farmaazon** [comment]
Why is it `x-on-hold`? Do we wait for some other task/design to be able to start this?

**Comment by somebody1234** [comment]
@farmaazon no, tbh I'm not completely sure why i added that tag, i've removed it now
